src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -1198,18 +1198,27 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 		return executeTaskResult;
 	}
 
+	private taskLabel(task: Task): string {
+		let label = task._label;
+		const workspaceFolder = task.getWorkspaceFolder();
+		if (workspaceFolder) {
+			label += ' (' + workspaceFolder.name + ')';
+		}
+		return label;
+	}
+
```

Comment:
This is the same string that appears in the tasks panel while a task is running, so I assume there's already a function that produces it somewhere - but after a cursory search I didn't find it.

If there *is* a suitable function that could be used instead of taskLabel, please let me know...

Reply:
thanks so much for looking into this. There's a `task.getQualifiedLabel()` function that does pretty much this. We should use that instead